### PR TITLE
ticdc: update scale out config

### PR DESCRIPTION
**Pull request by overvenus**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update TiCDC scale out config.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -254,6 +254,9 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 enable-table-across-nodes = true
 # 打开该功能后，该功能只对 Region 个数大于 `region-threshold` 值的表生效。
 region-threshold = 100000
+# 打开该功能后，该功能会对每分钟修改行数大于 `write-key-threshold` 值的表生效。
+# 注意：该参数默认值为 0，代表该功能默认不会按修改行数来划分表。
+write-key-threshold = 0
```
Comment (nongfushanquan) [inline ticdc/ticdc-sink-to-kafka.md:259]: 1）示例的值建议是一个合理值，如 30000，同时注明默认值为 0 。2) 说明用户设置了两个参数值后的行为。

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -254,6 +254,9 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 enable-table-across-nodes = true
```
Comment (nongfushanquan) [inline ticdc/ticdc-sink-to-kafka.md:254]: 说明默认值为 false

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -250,10 +250,18 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
 # 打开该功能后，该功能只对 Region 个数大于 `region-threshold` 值的表生效。
 region-threshold = 100000
+# 打开该功能后，该功能会对每分钟修改行数大于 `write-key-threshold` 值的表生效。
+# 注意：
+# * 该参数默认值为 0，代表该功能默认不会按修改行数来划分表。
+# * 用户可以根据集群负载来配置该参数，如 30000，代表该功能将分裂每分钟更新行数超过 30000 行的表。
+# * 当 `region-threshold` 和 `write-key-threshold` 同时配置时，
+#   该功能将优先检查修改行数是否大于 `write-key-threshold`，
+#   如果不超过，则再检查 Region 个数是否大于 `region-threshold`。
+write-key-threshold = 0
```
Comment (overvenus) [inline ticdc/ticdc-sink-to-kafka.md:264]:
```suggestion
write-key-threshold = 30000
```

File: ticdc/ticdc-changefeed-config.md [hunk 4]
```diff
@@ -94,8 +94,11 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
 # 注意：该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
 # 默认为 "false"。设置为 "true" 以打开该功能。
 enable-table-across-nodes = false
-# 打开该功能后，该功能只对 Region 个数大于 `region-threshold` 值的表生效。
+# 打开该功能后，该功能会对 Region 个数大于 `region-threshold` 值的表生效。
 region-threshold = 100000
+# 打开该功能后，该功能会对每分钟修改行数大于 `write-key-threshold` 值的表生效。
+# 注意：该参数默认值为 0，代表该功能默认不会按修改行数来划分表。
```
Comment (overvenus) [inline ticdc/ticdc-changefeed-config.md:100]:
```suggestion
# 注意：
# * 该参数默认值为 0，代表该功能默认不会按修改行数来划分表。
# * 用户可以根据集群负载来配置该参数，如 30000，代表该功能将分裂每分钟更新行数超过 30000 行的表。
# * 当 `region-threshold` 和 `write-key-threshold` 同时配置时，
#   该功能将优先检查修改行数是否大于 `write-key-threshold`，
#   如果不超过，则再检查 Region 个数是否大于 `region-threshold`。```

**Review by nongfushanquan — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 5]
```diff
@@ -94,8 +94,16 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
 # 注意：该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
 # 默认为 "false"。设置为 "true" 以打开该功能。
 enable-table-across-nodes = false
-# 打开该功能后，该功能只对 Region 个数大于 `region-threshold` 值的表生效。
+# 打开该功能后，该功能会对 Region 个数大于 `region-threshold` 值的表生效。
 region-threshold = 100000
+# 打开该功能后，该功能会对每分钟修改行数大于 `write-key-threshold` 值的表生效。
+# 注意：
+# * 该参数默认值为 0，代表该功能默认不会按修改行数来划分表。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:101]:
```suggestion
# * 该参数默认值为 0，代表该功能默认不会按表的修改行数来切分表的同步范围。
```

File: ticdc/ticdc-changefeed-config.md [hunk 6]
```diff
@@ -94,8 +94,16 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
...
+# * 用户可以根据集群负载来配置该参数，如 30000，代表该功能将分裂每分钟更新行数超过 30000 行的表。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:102]:
```suggestion
# * 你可以根据集群负载来配置该参数，如 30000，代表当表每分钟的更新行数超过 30000 时，该功能将会切分表的同步范围。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 7]
```diff
@@ -237,7 +237,7 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 
 ## 横向扩展大单表的负载到多个 TiCDC 节点
 
-该功能通过将大单表按 Region 个数切分成多个数据范围，将这些数据范围分布到多个 TiCDC 节点上，使得多个 TiCDC 节点可以同时同步大单表。该功能可以解决以下两个问题：
+该功能通过将大单表按数据量和每分钟修改行数切分成多个数据范围，使得数据范围之间的数据量和每分钟修改行数基本相同。该功能将这些数据范围分布到多个 TiCDC 节点上，使得多个 TiCDC 节点可以同时同步大单表。该功能可以解决以下两个问题：
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:240]:
```suggestion
该功能可以按照大单表的数据量和每分钟的修改行数将表的同步范围切分为多个，并使各个范围之间所同步的数据量和修改行数基本相同。该功能将这些范围分布到多个 TiCDC 节点进行同步，使得多个 TiCDC 节点可以同时同步大单表。该功能可以解决以下两个问题：
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 8]
```diff
@@ -250,10 +250,18 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
 ```toml
 [scheduler]
-# 设置为 "true" 以打开该功能。
+# 默认值为 "false"，设置为 "true" 以打开该功能。
 enable-table-across-nodes = true
 # 打开该功能后，该功能只对 Region 个数大于 `region-threshold` 值的表生效。
 region-threshold = 100000
+# 打开该功能后，该功能会对每分钟修改行数大于 `write-key-threshold` 值的表生效。
+# 注意：
+# * 该参数默认值为 0，代表该功能默认不会按修改行数来划分表。
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:259]:
```suggestion
# * 该参数默认值为 0，代表该功能默认不会按表的修改行数来切分表的同步范围。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 9]
```diff
@@ -250,10 +250,18 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+# * 用户可以根据集群负载来配置该参数，如 30000，代表该功能将分裂每分钟更新行数超过 30000 行的表。
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:260]:
```suggestion
# * 你可以根据集群负载来配置该参数，如 30000，代表当表每分钟的更新行数超过 30000 时，该功能将会切分表的同步范围。
```

File: ticdc/ticdc-changefeed-config.md [hunk 10]
```diff
@@ -94,8 +94,16 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
...
+# * 当 `region-threshold` 和 `write-key-threshold` 同时配置时，
+#   该功能将优先检查修改行数是否大于 `write-key-threshold`，
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:104]:
```suggestion
#   TiCDC 将优先检查修改行数是否大于 `write-key-threshold`，
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 11]
```diff
@@ -250,10 +250,18 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+# * 当 `region-threshold` 和 `write-key-threshold` 同时配置时，
+#   该功能将优先检查修改行数是否大于 `write-key-threshold`，
```
Comment (qiancai) [inline ticdc/ticdc-sink-to-kafka.md:262]:
```suggestion
#   TiCDC 将优先检查修改行数是否大于 `write-key-threshold`，
```

**Review by qiancai — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 12]
```diff
@@ -94,8 +94,16 @@ ignore-insert-value-expr = "price > 1000 and origin = 'no where'" # 忽略包含
 # 注意：该功能只在 Kafka changefeed 上生效，暂不支持 MySQL changefeed。
 # 默认为 "false"。设置为 "true" 以打开该功能。
 enable-table-across-nodes = false
-# 打开该功能后，该功能只对 Region 个数大于 `region-threshold` 值的表生效。
+# 打开该功能后，该功能会对 Region 个数大于 `region-threshold` 值的表生效。
 region-threshold = 100000
+# 打开该功能后，该功能会对每分钟修改行数大于 `write-key-threshold` 值的表生效。
+# 注意：
+# * 该参数默认值为 0，代表该功能默认不会按表的修改行数来切分表的同步范围。
```
Comment (qiancai) [inline ticdc/ticdc-changefeed-config.md:101]:
```suggestion
# * `write-key-threshold` 参数默认值为 0，代表该功能默认不会按表的修改行数来切分表的同步范围。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 13]
```diff
@@ -250,10 +250,18 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
 # 打开该功能后，该功能只对 Region 个数大于 `region-threshold` 值的表生效。
 region-threshold = 100000
+# 打开该功能后，该功能会对每分钟修改行数大于 `write-key-threshold` 值的表生效。
+# 注意：
+# * 该参数默认值为 0，代表该功能默认不会按表的修改行数来切分表的同步范围。
+# * 你可以根据集群负载来配置该参数，如 30000，代表当表每分钟的更新行数超过 30000 时，该功能将会切分表的同步范围。
+# * 当 `region-threshold` 和 `write-key-threshold` 同时配置时，
+#   TiCDC 将优先检查修改行数是否大于 `write-key-threshold`，
+#   如果不超过，则再检查 Region 个数是否大于 `region-threshold`。
+write-key-threshold = 30000
```
Comment (lilin90) [inline ticdc/ticdc-sink-to-kafka.md:264]: 这里是特意写的 30000 吗？是否需要跟 `ticdc/ticdc-changefeed-config.md` 文档里的 0 保持一致？

Comment (qiancai) [reply ticdc/ticdc-sink-to-kafka.md:264]: @overvenus PTAL

Comment (overvenus) [reply ticdc/ticdc-sink-to-kafka.md:264]: 特意写的，这个是推荐配置，0 是默认配置

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**